modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/PublisherCommentTest.java

```diff
@@ -141,6 +147,7 @@ public void testPublisherCommentTest() throws Exception {
             String replyCommentId = addedCommentDTO.getId();
             assertNotNull(replyCommentId, "Comment Id is null");
             replies.add(replyCommentId);
+            Thread.sleep(1000);
```

Comment:
Let's add a comment saying why this sleep is added. Let's do the same in other places as well.

Reply:
Added a common comment at the beginning of the test case
![comments](https://user-images.githubusercontent.com/30455603/112785291-49469a00-9071-11eb-8065-cf8cfc7559d0.png)

Reply:
Yeah. That is fine